_includes/releases/v22.2/v22.2.0-beta.6.md

```diff
@@ -0,0 +1,28 @@
+## v22.2.0-beta.6
+
+Release Date: November 7, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-beta-6-sql-language-changes">SQL language changes</h3>
+
+- Fixed a bug that could cause crashes when parsing malformed [changefeed expressions](../v22.2/cdc-transformations.html). [#90844][#90844]
+- Added the boolean column `implicit_txn` to `crdb_internal.cluster_execution_insights` and `crdb_internal.node_execution_insights`. [#90872][#90872]
+
+<h3 id="v22-2-0-beta-6-bug-fixes">Bug fixes</h3>
+
+- TTL decoding error messages now correctly contain hex-encoded key bytes instead of hex-encoded key pretty-print output. [#90726][#90726]
+- Remove redundant assertion that `ExportRequests` should have the parameter `ReturnSST`. [#90833][#90833]
+- Fixed a bug that resulted in some retriable errors not being retried during [`IMPORT`](../v22.2/import.html). [#90429][#90429]
+- Fixed a bug that caused incorrect results and internal errors when a [`LEFT JOIN`](../v22.2/joins.html) operated on a table with virtual computed columns. The bug only presented when the optimizer planned a "paired joiner" and could cause values of the virtual columns to be incorrectly `NULL`. An internal error would occur in the same situation if the virtual column had a `NOT NULL` constraint. [#90999][#90999]
```

Comment:
"optimizer" could link to https://www.cockroachlabs.com/docs/v22.2/cost-based-optimizer.html 

"virtual computed columns" could link to https://www.cockroachlabs.com/docs/v22.2/computed-columns.html

"not null constraint" could link to https://www.cockroachlabs.com/docs/v22.2/not-null.html